### DISCUSSION TITLE
React Native 0.67

### DISCUSSION THREAD

**Discussion by iMaupin** [wix/Detox#3217] [Q&A] [answered]
With the release of 0.67 React-Native is bumping to Gradle 7.2. Is detox currently compatible with this version of Gradle?

**Comment by asafkorem** [comment, marked as answer]
@iMaupin I'm not sure if I understand your question, but I'll still try to answer (please tell me if it still does not answer your question). 😅 

Since RN v.67 is compatible with this version of Gradle, your project should be compatible with this Gradle version (if you're using RN v.67). 
Detox' Gradle version is untied with the version of RN since it does not depend on it, so you should be able to execute Detox on RN v.67 projects, with Gradle v7.2.

Our example / test project is running with Gradle v6.9 and we'll probably keep it after we'll upgrade its [RN version](https://github.com/wix/Detox/pull/3202) from CI related reasons, but this will not affect your project.

**Reply by asafkorem** [reply]
@jonathanmos please CMIIW 🙂

**Reply by iMaupin** [reply]
@asafkorem awesome. I assumed that would be the case, we are just in the process of updating to 0.67 and wanted to make sure that this would not affect our test suite. Thank you for clarifying!
